### PR TITLE
chore(CX-2268): clean up feature flags AREnableMyCollectionAndroid and AREnableMyCollectionIOS

### PR DESCRIPTION
**Pull request by dariakoko**
The type of this PR is: **Enhancement**

This PR resolves [CX-2268]

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/main/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/main/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [ ] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- Removed released feature flags: AREnableMyCollectionAndroid and AREnableMyCollectionIOS. Followed by code cleanup. - Daria

</details>

[CX-2268]: https://artsyproduct.atlassian.net/browse/CX-2268?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ❌ **An error occurred while validating your changelog, please make sure you provided a valid changelog.**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against b3729440ac5f2d9335bbc54e297d28c530e8b08d
</p>

File: src/lib/Scenes/MyProfile/MyProfile.tsx [hunk 1]
```diff
@@ -1,209 +1,15 @@
 import { OwnerType } from "@artsy/cohesion"
```
Comment (olerichter00) [inline src/lib/Scenes/MyProfile/MyProfile.tsx:1]: Do you think we can remove this file completely? It is now just wrapping `MyCollectionAndSavedWorksQueryRenderer` and we could move the screen tracking in line 10 to `MyCollectionAndSavedWorks.tsx`.

File: src/lib/Scenes/Home/Home.tsx [hunk 2]
```diff
@@ -76,7 +76,6 @@ const Home = (props: Props) => {
 
   useEffect(() => {
     prefetchUrl("search")
-    prefetchUrl("my-profile")
```
Comment (olerichter00) [inline src/lib/Scenes/Home/Home.tsx:79]: In this effect we are prefetching the queries for all screens that are accessible via the bottom navigation. Do you think we can instead prefetch the query `MyCollectionAndSavedWorksQuery`? 

Let me know if you have questions about how prefetching is implemented in Eigen. We can work on this part together if you like.

**Review by olerichter00 — COMMENTED** [review]
Thanks a lot, this is great work 🎉 

I've just added a few comments about `MyProfile.tsx` and prefetching.

File: src/lib/Scenes/Home/Home.tsx (hunk 2 above)
Comment (dariakoko) [reply src/lib/Scenes/Home/Home.tsx]: Done! Thanks for help :)

**Review by olerichter00 — APPROVED** [review]
Feel free to merge as soon as CircleCI is green 🍏

**Pull request merged**